PocketKit/Sources/PocketKit/Services.swift

```diff
@@ -115,7 +115,7 @@ struct Services {
             userDefaults: userDefaults,
             badgeProvider: UIApplication.shared
         )
-        subscriptionStore = PocketSubscriptionStore(user: user, receiptService: AppStoreReceiptService(client: Services.shared.v3Client))
+        subscriptionStore = PocketSubscriptionStore(user: user, receiptService: AppStoreReceiptService(client: v3Client))
```

Comment:
@Gio2018 FYI had to change this, there was a circular depedency. Curious how your tests passed.